example/split-channels/index.html

```diff
@@ -70,6 +70,25 @@ <h3>How to Enable Split Channels</h3>
     container: document.querySelector('#wave'),
     splitChannels: true
 });
+</code></pre>
+                </p>
+                <h3>How to use different colors per channel with Split Channels</h3>
+                <p>
+                    With the <code>splitChannels</code> option set to <code>true</code>, you can specify different colors for the second channel if desired.
+                </p>
+                <p>
+                    To set these colors, use the <code>channelColors</code> property defined with <code>waveColors</code> and <code>progressColors</code>.
+                </p>
+                <p>
+<pre><code>
+    const wavesurfer = WaveSurfer.create({
+        container: document.querySelector('#wave'),
+        splitChannels: true,
+        channelColors: {
+            waveColors: ['red', 'green'],
```

Comment:
can you also use a hex color value here to demonstrate that works as well?

Reply:
Absolutely.